contracts/sophia.md

```diff
@@ -59,6 +59,40 @@ worked out).
 
 *NOTE: Contract inheritance is not yet implemented.*
 
+#### Calling other contracts
+
+To call a function in another contract you need the address to an instance of
+the contract. The type of the address is a (possibly abstract) contract. For
+instance, given the `VotingType` contract above we can define a contract
+
+```javascript
+contract VoteTwice =
+  public function voteTwice(v : VotingType, alt : string) =
+    v.vote(alt)
+    v.vote(alt)
+```
+
+Contract calls take two optional named arguments `gas : int` and `value : int`
+that lets you set a gas limit and provide tokens to a contract call. If omitted
+the defaults are no gas limit and no tokens. Suppose there is a fee for voting:
+
+```javascript
+  function voteTwice(v : VotingType, fee : int, alt : string) =
+    v.vote(value = fee, alt)
```

Comment:
If I understand this correctly, before this line I would expect check that `(2 * fee) =< Call.value`. Or a comment mentioning not to copy-paste this code in production?